src/scss/index.css

```diff
@@ -164,7 +164,7 @@ input[type=password] {
     display: none !important;
   }
 }
-#navItems:hover {
+#navLink:hover {
   color: black !important;
 }
 
```

Comment:
I am confused by this, #navItems is changed to #navLink, but no elements in the code has an ID of navLink

From my understanding the duplicated #navItems ID's were used to apply this style to all the buttons. Since we dont want duplicate ID's, you gave the elements new ID's. 

At this point, this style should be applied using the class nav-link which is applied to all the elements.
This is also the case, and being done src/scss/custom/_header.scss.

Am I missing something?

Reply:
Its a typo! It shuld be : .nav-link. 
The a tags has class of nav-links, u put the correct tag in the _header.scss file.

I will correct the error tomorrow morning

Reply:
Allright, perfect!